app/views/members/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1> Members </h1>
+
+<% if can? :manage, User %>
+  <td><%= link_to "Add new", new_member_path %></td>
+<% end %>
+
+<table>
+  <% @members.each do |member| %>
+    <tr>
+      <td><%= "#{member.first_name} #{member.last_name}" %></td>
+      <% if can? :manage, member %>
```

Comment:
no need for this line when you are explicitly checking the ability of each action